portal/config/eproms/Questionnaire.json

```
@@ -6965,6 +6965,15 @@
         {
           "type": "choice",
           "linkId": "ironman_ss.1",
+          "code":  {
```

I think the value of `code` should be an array/list of objects, going from the [STU3 example](http://hl7.org/implement/standards/fhir/STU3/questionnaire-example.json)
```
  "item": [
    {
      "linkId": "1",
      "code": [
        {
          "system": "http://example.org/system/code/sections",
          "code": "COMORBIDITY"
        }
      ],
```


```suggestion
          "code":  [
```

---

@ivan-c  done, see my latest commit here.